pkg/build/pipelines/python/build-gpep517.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build a Python wheel with gpep517
+
+needs:
+  packages:
+    - busybox
+
+pipeline:
+  - runs: |
+      if ! [ -x "$(command -v python3)" ]; then
```

Comment:
how about adding this in `needs.packages`? I think you also want `py3-gpep517` there as well?